Frends.AmazonS3.DownloadObject/Frends.AmazonS3.DownloadObject.Tests/UnitTest1.cs

```diff
@@ -0,0 +1,554 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.IO;
+using System.Linq;
+using Amazon.S3;
+using Amazon;
+using Amazon.S3.Model;
+
+namespace Frends.AmazonS3.DownloadObject.Tests
+{
+    [TestClass]
+    public class UnitTest1
+    {
+        private readonly string? _accessKey = Environment.GetEnvironmentVariable("HiQ_AWSS3Test_AccessKey");
+        private readonly string? _secretAccessKey = Environment.GetEnvironmentVariable("HiQ_AWSS3Test_SecretAccessKey");
+        private readonly string? _bucketName = Environment.GetEnvironmentVariable("HiQ_AWSS3Test_BucketName");
+        private readonly string _dir = Path.Combine(Environment.CurrentDirectory); // .\Frends.AmazonS3.DownloadObject\Frends.AmazonS3.DownloadObject.Test\bin\Debug\net6.0\
+
+        Connection? _connection;
+        Input? _input;
+        Options? _options;
+
+
+        /// <summary>
+        /// Create test files and folders. 
+        /// !!! Manually upload test files to S3 "2022/"-folder. You can copy files from ".\Frends.AmazonS3.DownloadObject\Frends.AmazonS3.DownloadObject.Test\bin\Debug\net6.0\DownloadTestFiles" after first test run. !!!
+        /// </summary>
+        [TestInitialize]
+        public void Initialize()
+        {
+            CreateTestFiles();
+        }
+
+        #region pre-signed URL
```

Comment:
No need to regions here.

Reply:
Tests are in 2 parts, pre-signed and AWS creds, so it's easier to follow the code when you can collapse pre-signed part out of sight. This doesn't affect the code.